src/DsaLocal.cc

```diff
@@ -731,7 +732,18 @@ void IntraBlockBuilder::visitCallSite(CallSite CS) {
              << " isResolved: " << dsaCallSite->isResolved() << "\n");
 
   Instruction *inst = CS.getInstruction();
+  LOG("dsa-callsite", errs()<<"Inst:"<<inst->getName()<<"\n");
+  //TODO: not clear if we should create a new node or find an existing one in local graph
```

Comment:
This is the part of the local pass that creates nodes for all the pointers. 
It should already have code for creating nodes for the function pointer being called.
The only thing that should be required for you is to create the callsite (which you already do).

The incomplete marking cannot be done at this point. You know that a node is incomplete if it is reachable from arguments or flows out through a return. This information is only available at the end of the pass.